src/jmh/java/benchmark/AssertBenchmark.java

```diff
@@ -18,7 +18,7 @@
 
 @Warmup(iterations = 2, time = 5, batchSize = 50)
 @Measurement(iterations = 3, batchSize = 50)
-@Fork(3)
```

Comment:
Fix offending tests, even though these are in the `benchmark` package, and not `performance` (used by the automated pipeline)